test/core/security/tsi_auth_context_util.cc

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ * Copyright 2019 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "test/core/security/tsi_auth_context_util.h"
+
+#include <grpc/grpc.h>
+#include <grpc/support/alloc.h>
+#include <grpc/support/log.h>
+
+#include "src/core/lib/security/context/security_context.h"
+
+bool test_identity_from_auth_context(const grpc_auth_context* ctx,
```

Comment:
Is this API use in tests only? If yes, please change the file name to include the word "test" otherwise maybe replace "test" with "check" in the name.

Reply:
Good call. Renamed it `check_identity_from_auth_context_for_testing`.